cache/cache.go

```diff
@@ -48,12 +48,22 @@ func NewRandomEvictedCache(maxBytes int) Cache {
 	}
 }
 
+func (c *randomEvictedCache) entrySize(key *string, value Measurable) int {
```

Comment:
Yeah it's immutable, so there's no "write". But for the same reason it's not copied, so still good point!